cmd/meroxa/turbine_cli/utils.go

```diff
@@ -308,3 +311,58 @@ func RunCmdWithErrorDetection(ctx context.Context, cmd *exec.Cmd, l log.Logger)
 	l.Info(ctx, successMsg)
 	return successMsg, nil
 }
+
+// CreateTarAndZipFile creates a .tar.gz file from `src` on current directory.
+func CreateTarAndZipFile(src string, buf io.Writer) error {
+	// Grab the directory we care about (app's directory)
+	appDir := filepath.Base(src)
+
+	// Change to parent's app directory
+	err := os.Chdir(filepath.Dir(src))
```

Comment:
to be consistent, this function could cd back to pwd like some of the other functions